components/Sidebar.tsx

```diff
@@ -61,11 +61,11 @@ const Sidebar = () => {
 
       {/* footer attribution */}
       <div className="text-xs text-white/50">
-        JunoTools 1.0. Copyright &copy; {new Date().getFullYear()} by{' '}
+        JunoTools 0.1.0-beta <br />
```

Comment:
How about getting the version from `package.json`?

Reply:
I like this idea @grikomsn — any ideas how we would accomplish that?

Reply:
- https://github.com/CosmosContracts/juno-tools/pull/47